README.md

```diff
@@ -84,7 +84,7 @@ action-validator .github/workflows/build.yml
 
 Use `action-validator -h` to see additional options.
 
-> ### CAUTION
+> ### ⚠️ CAUTION
```

Comment:
```suggestion
> ### CAUTION
```

I'm not a huge fan of emoji in READMEs; too many situations don't handle them very well.

Reply:
I was just trying to make it consistent with https://github.com/mpalmer/action-validator#node-api and all the other emojis in code blocks. ;)

Reply:
Yeah, they're a bit different, because they're just dumps of program output -- typically they're just copy-pasta'd from a run of the program, we're not likely to edit them.  But in places where a new contributor may have to roam, and that person's editor doesn't deal well with them, it can get a bit ugly.